handlers/conceptHandlers.go

```diff
@@ -24,7 +24,17 @@ func GetConceptUpdatesHandler() func(c *gin.Context) {
 			size = 10
 		}
 
-		concepts, status := service.GetConceptUpdates(c.Request.Context(), conceptId, page, size)
+		sortBy := c.Query("sort_by")
+		if sortBy == "" {
+			sortBy = "datetime"
+		}
+
+		sortOrder, err := strconv.Atoi(c.Query("sort_order"))
+		if err != nil {
+			sortOrder = -1 // descending order by default
```

Comment:
Ta inn sort_order til å ta inn string-verdi, som har enum asc & desc og map det om til tilsvarende int-verdi i service elns
```suggestion
			sortOrder = "desc" // descending order by default
```